javascript/chronometer.js

```diff
@@ -1,26 +1,53 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = "";
+    //this.milliseconds = 0;
+    //this.intervalIdMilli = "";
   }
   startClick(callback) {
-    // ... your code goes here
-  }
+    this.intervalId = setInterval(() => {
+      this.currentTime += 1
+      callback();
+    }, 1000)
+    // this.intervalId = setInterval(() => {
```

Comment:
for the Bonus to work propertly there were two things breaking the code
1. Here you needed to save this interval inside the intervalIdMilli attribute, not intervalId, as that one is used for the first interval. This will overwrite the value and cause issues when trying to clear them
2. See line 81.